merlin-server/sql/merlin/tables/constraint.sql

```diff
@@ -84,22 +84,19 @@ for each row
 execute function constraint_set_updated_at();
 
 create function constraint_check_constraint_run()
-returns trigger
-security definer
-language plpgsql as $$begin
-  if new.definition != old.definition then
-    update constraint_run
-    set definition_outdated = true
-    where constraint_id = new.id
-      and constraint_definition != new.definition
-      and definition_outdated = false;
-  else
-    update constraint_run
-    set definition_outdated = false
-    where constraint_id = new.id
-      and constraint_definition == new.definition
-      and definition_outdated = true;
-  end if;
+  returns trigger
+  security definer
+  language plpgsql as $$begin
+  update constraint_run
```

Comment:
Minor: readability:

```suggestion
  returns trigger
  security definer
  language plpgsql as $$
begin
  update constraint_run
```

Reply:
made the change, my fingers are crossed I didn't break the test